_posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Six lines to install and start SparkR on Mac OS X Yosemite
+excerpt: for both  and command line
+categories: articles
+tags: [R, Spark, SparkR, Hadoop, big data]
+comments: true
+share: true
+published: true
+author: yuki
+---
+
+<span class = "dropcap">I</span> know there are many R users who like to test out SparkR without all the configuration hassle. Just these six lines and you can start SparkR from both RStudio and command line. 
+
+![luigi-workflow]({{ site.url }}/images/spark-logo.png)
+
+
+# One line for Spark and SparkR
+
+> Apache Spark is a fast and general-purpose cluster computing system
+
+
+> SparkR is an R package that provides a light-weight frontend to use Apache Spark from R
+
+# Six lines to start SparkR
+The first three lines should be called in your command line.
+{% highlight sh %}
+brew update # If you don't have homebrew, get it from here (http://brew.sh/)
```

Comment:
homebrew is the de facto mac package manager.
i didnt test apt-get so not this time.